README.md

```diff
@@ -1,6 +1,6 @@
 # cadence-docs
 
 ## Setting up for local development
-This will start a local server and can be accessed at http://localhost:8081/
+This will start a local server and can be accessed at http://localhost:8080/
```

Comment:
do update the port to 8081? I remembered we used 8081?

Reply:
I think the vuepress library auto updated in a minor release. I tested this locally and found that it has switched port as well. Thanks Long for updating the docs for this one.